database/postgres/postgres_game.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict
+from datetime import datetime
+from database.models import Games
+from database.postgres.postgres_country import save_country
+from database.postgres.postgres_team import save_team
+from database.postgres.postgres_league import save_league
+
+def save_games(json_data: Dict, sport_id, session) -> None:
+    games = json_data.get("response", [])
+    if not games:
+        print('insufficient data')
+        return
+
+    for game in games:
+        if "fixture" in game or "game" in game:
+            separated_data = game.get("fixture") or game.get("game")
+            api_id = separated_data.get("id")
+            date = separated_data.get("date")
+            status = separated_data.get('status')
+        else:
+            api_id = game.get('id')
+            date = game.get('date')
+            status = game.get('status')
+
+        country_data = game.get("location").get("country") if "location" in game else game.get("country")
```

Comment:
We will replace it with a model/class anyway, but:

whenever you do something like here you MUST first verify whether such keys are present in your dictionary. What would happen if dictionary did not contain "country" within it? 

Our code has to be as "bulletproof" as possible